src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputEmptyLineSeparatorFormerException.java

```diff
@@ -1,6 +0,0 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
-
-public class InputEmptyLineSeparatorFormerException
-{
-
-}
```

Comment:
@romani Aha. Got it. I executed the first command on all my files. But the second command doesnt seem to be right (I apologize if I am wrong) because command git <oldname> <newname>, git assumes <oldname> to be a command.